src/read.c

```diff
@@ -2566,10 +2566,8 @@ void ReadTryNext (
 
 void ReadHelp(TypSymbolSet follow)
 {
-    Obj topic;
-
-    C_NEW_STRING(topic, STATE(ValueLen), (void *)STATE(Value));
-    TRY_READ { IntrHelp(topic); };
+    TRY_READ { IntrHelp(STATE(ValueObj)); }
+    STATE(ValueObj) = 0;
```

Comment:
Why clean ValueObj?

Reply:
Paranoia, I guess: The idea is that we already used this string in `IntrHelp`, and don't want to accidentally reuse it elsewhere (say, due to a big, where some function expects the scanner to set `ValueObj`, but this doesn't happen -- either due to a bug in the scanner, or because the calling code was n fact wrong to assume the scanner would set `ValueObj`).

 In any case, I have another change coming up in another PR where I reset `ValueObj` and `Value` at the start of `Match`, which would make this change partially obsolete (but not generally, I guess).

Anyway, if you are concerned about it now, I can get rid of it before merging.